### PR TITLE
Fix eth-lightwallet.js

### LINKED ISSUE #20: xrpk wallet fails with "window is not defined"
Running `xrpk wallet` gives the following error:

```bash
$ xrpk wallet       
seed phrase (BIP39 format, default: auto):
ReferenceError: window is not defined
    at Function.Random.getRandomBufferBrowser (/mnt/c/Users/Shubham/Documents/GitHub/MLH/xrpackage-cli/eth-lightwallet.js:9025:3)
    at Function.Random.getRandomBuffer (/mnt/c/Users/Shubham/Documents/GitHub/MLH/xrpackage-cli/eth-lightwallet.js:9015:38)
    at Function.Mnemonic._mnemonic (/mnt/c/Users/Shubham/Documents/GitHub/MLH/xrpackage-cli/eth-lightwallet.js:23227:20)
    at new Mnemonic (/mnt/c/Users/Shubham/Documents/GitHub/MLH/xrpackage-cli/eth-lightwallet.js:23060:31)
    at Function.KeyStore.generateRandomSeed (/mnt/c/Users/Shubham/Documents/GitHub/MLH/xrpackage-cli/eth-lightwallet.js:465:12)
    at Object.yargs.scriptName.command.command.command.command [as handler] (/mnt/c/Users/Shubham/Documents/GitHub/MLH/xrpackage-cli/cli.js:820:41)
    at process._tickCallback (internal/process/next_tick.js:68:7) 
```

It looks like it's stemming from the `Random.getRandomBufferBrowser` method of [`eth-lightwallet.js`](https://github.com/webaverse/xrpackage-cli/blob/master/eth-lightwallet.js), which looks like it assumes a browser environment, I think:
```js
Random.getRandomBufferBrowser = function (size) {
  if (!window.crypto && !window.msCrypto) throw new Error('window.crypto not available');
  if (window.crypto && window.crypto.getRandomValues) var crypto = window.crypto;else if (window.msCrypto && window.msCrypto.getRandomValues) //internet explorer
    var crypto = window.msCrypto;else throw new Error('window.crypto.getRandomValues not available');
  var bbuf = new Uint8Array(size);
  crypto.getRandomValues(bbuf);
  var buf = Buffer.from(bbuf);
  return buf;
};
```

### PR DESCRIPTION
**Pull request by shu8**
Generated using Browserify:

```
npm install eth-lightwallet
cd node_modules/eth-lightwallet
browserify index.js -o eth-lightwallet.js --node --standalone ethlightwallet
mv eth-lightwallet.js ../../eth-lightwallet.js
```

and then manually going through the file to update usages of the `Buffer` API to the new one, mostly going through all occurrences of

- `new Buffer`,
- `buffer.Buffer`

and deciding which new API is appropriate (`Buffer.alloc` for numbers or `Buffer.from` for strings/objects). This seems to have fixed all the warnings when running with Node's `--trace-deprecation` flag.

Fixes #20.

CLI tool commands like `wallet` now working:

image

### CODE REVIEW

**Comment by avaer** [comment]
Oh nice! This seems like the right fix.

Since this is a weird procedure to generate the `eth-lightwallet.js`, it might be worth it to document/include the procedure somewhere in the repo -- pehaps a bash script, or a comment in the generated file, or both?

**Comment by shu8** [comment]
Thanks! Added:

- a note to README
- a note to the generated `eth-lightwallet.js` file
- a quick bash script to generate the file
- `browserify` as a dev dependency

**Comment by shu8** [comment]
Fixed merge conflicts 👍

File: gen-eth-lightwallet [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+echo "Performing bundling"
+browserify index.js -o eth-lightwallet.js --node --standalone ethlightwallet
+
+echo "Replacing repo eth-lightwallet.js file with new bundled file"
+mv eth-lightwallet.js ../../eth-lightwallet.js
+cd ../../
+
+echo "Uninstalling eth-lightwallet"
+npm remove eth-lightwallet
```
Comment (avaer) [inline gen-eth-lightwallet:14]: Not sure if this is a good idea; it could be a destructive operation. I think it's fine to leave the module.

**Pull request merged**